ergo/questions/binary.py

```diff
@@ -0,0 +1,33 @@
+from abc import abstractmethod
+
+import requests
+
+from ergo.distributions.base import flip
+
+from .question import Question
+
+
+class BinaryQuestion(Question):
+    @abstractmethod
+    def get_community_prediction(self) -> float:
+        """
+        Get the latest community probability for the binary event
+        """
+        raise NotImplementedError("This should be implemented by a subclass")
+
+    @abstractmethod
+    def submit(self, p: float, confidence: float = 0) -> requests.Response:
```

Comment:
What is the difference between `p` and `confidence`? Does it extend to all prediction platforms?

Reply:
Yeah, this was the abstraction I was least confident in (pun slightly intended). For both Almanis and PredictIt, you both submit a prediction to the platform, and then some number of points/dollars to "bet" on that prediction. That was the idea of confidence. This wouldn't apply for Foretold or Metaculus predictions.